core/src/main/java/com/equo/application/client/EquoApp.java

```diff
@@ -101,7 +101,8 @@ public static void setAppName(String appName) {
         appId = appId.substring(0, appId.length() - 1);
       }
     }
-    APP_ID = appId;
+    APP_ID = CUSTOM_URL = appId;
+    CLASSPATH_URI = String.format("%s://%s/", CLASSPATH_SCHEME, CUSTOM_URL);
```

Comment:
You could remove the variable, which I think doesn't make sense anymore, and create a function.
Before, it made sense because it was a constant.